doc/release-notes.md

```diff
@@ -31,7 +31,66 @@ Example item
 
 Example text.
 
-0.12.1 Change log
+
+dash-cli: arguments privacy
+--------------------------------
+
+The RPC command line client gained a new argument, `-stdin`
+to read extra arguments from standard input, one per line until EOF/Ctrl-D.
+For example:
+
+    $ echo -e "mysecretcode\n120" | src/dash-cli -stdin walletpassphrase
+
+It is recommended to use this for sensitive information such as wallet
+passphrases, as command-line arguments can usually be read from the process
+table by any user on the system.
+
+RPC low-level changes
+----------------------
+
+- `gettxoutsetinfo` UTXO hash (`hash_serialized`) has changed. There was a divergence between
```

Comment:
References https://github.com/bitcoin/bitcoin/pull/7848, which is also not in Dash yet